components/infobox/wikis/starcraft2/infobox_team_custom.lua

```diff
@@ -132,6 +131,12 @@ function CustomTeam:createBottomContent()
 	end
 end
 
+function CustomTeam:addToLpdb(lpdbData)
+	lpdbData.earnings = _EARNINGS
+	Variables.varDefine('team_name', lpdbData.name)
```

Comment:
This seems like it should be added elsewhere

Reply:
just removed it
idk why we ever set that var, it isn't used anywhere on the player page (just checked)